source/manual/data-gov-uk-continuous-integration-and-deployment.html.md

```diff
@@ -13,41 +13,34 @@ review_in: 3 months
+Each Find PR app is linked to a master Heroku Elasticsearch addon, while each Publish PR app has it's own isolated ES addon for testing against.
 
-The [Find Data][find] repository is linked to a [Heroku](#heroku-review-apps) account which generates apps for each PR opened.
+There is also a permanent instance of [Publish](https://datagovuk-publish.herokuapp.com/) and [Find](https://datagovuk-find.herokuapp.com/) running code on the master branches, which serve as a kind of integration environment (including GOV.UK Signon).
 
-See [Common tasks][common-tasks] for how to deploy to production.
-
-## Deploying Publish Data Worker
+GOV.UK Signon, Zendesk support tickets and Sentry are disabled for Heroku apps, with the default user for [Publish Data][publish] being assigned to the GDS organisation.
 
-While the [above](#deploying-find-data-and-publish-data) applies to both [Find Data][find] and [Publish Data][publish], Publish Data is in fact two apps; Publish Data itself and `publish-data-worker`. Both app share the same code in the [Publish Data repo][publish] but run off different manifest files in the repo. In the Publish Data app the manifest file runs the default rails server command, but in the worker app manifest `bundle exec sidekiq` is run instead.
-
-You can monitor Sidekiq jobs for the worker here: [https://publish-data-beta-staging.cloudapps.digital/sidekiq/]().
+## Deploying Find Data and Publish Data
 
-The release tag method described in [Common tasks][common-tasks] deploys both apps in this repository. See the relevant section in the [Find Data][find] `travis.yml`.
+Deployment of the [Find Data][find] and [Publish Data][publish] applications to integration and staging is done automatically when the master branch is modified.
 
-## Zero-downtime deployment
+Each app repo has a manifest file to set the environment variables and link to PaaS services. The manifests are used by the deploy scripts, which use cloudfoundry’s autopilot module.
 
-Populating the Elasticsearch service is done via a rake task in the [Publish Data][publish] app. It creates a new index, imports data from Elasticsearch, and changes the `datasets-[env]` alias to point to the new index. Since [Find Data][find] connects to Elasticsearch through that alias, the switchover to the new index is instant. This method is detailed in [Elasticsearch’s documentation on zero-downtime](https://www.elastic.co/guide/en/elasticsearch/guide/current/index-aliases.html#index-aliases).
+See [Common tasks][common-tasks] for how to deploy to production.
 
-## Heroku review apps
+## Deploying Publish Data Worker
 
-A Heroku review app is created for each PR opened and remains active while the PR is open. Speak to a member of the data.gov.uk team to get the account’s credentials.
+Publish Data is in fact two apps: the web app itself and `publish-data-worker`. Both app share the same code in the [Publish Data repo][publish] but run off different manifest files in the repo.
```

Comment:
app => apps

Reply:
Done - for my reputation I'd like to point out these were there before ;-)